Featur/settings menu functionality

ZeMelonCode opened the pull request.

ZeMelonCode committed:
Settings menu

Settings menu in BP

ZeMelonCode committed:
Merge branch 'develop' into featur/Settings-Menu-Functionality

ZeMelonCode committed:
Merged settings and menu Widget.

Merged Settings widget to Main Menu widget.

Removed Settings widget and Settings level map.

Added functionality for Settings menu

ZeMelonCode committed:
Funtionalities

Settings button brings you to settings.
Back button brings you back to main menu

ZeMelonCode committed:
SettingsWidget

Created Settings widget separated from Main menu widget

ZeMelonCode committed:
Merge branch 'develop' into featur/Settings-Menu-Functionality

ZeMelonCode committed:
Updated Settings Menu Navigation

Created Menu Manager
Fixed SettingsMenu bp Class
Added button functions to SettingsMenuWidget.cpp
Added Navigation from mainMenu to Settings menu and back

Bleep-Bloop committed:
Update WBP_MainMenuWidget.uasset

Made Button text a child of the button to prevent it from blocking clicks.

Bleep-Bloop commented:
On the MainMenuWidget I made the button text components children of their respective buttons to fix an issue with blocking clicks.

Bleep-Bloop committed:
PR Cleanup

Erased unnecessary comment in OnBackButtonPressed().
Removed unnecessary includes.
Cleaned up whitespace.
